### ISSUE TITLE
Abscent slots file-preview-before and file-preview-after

### ISSUE THREAD

**Issue by marinaglancy** [safrazik/vue-file-agent#112]
**Describe the bug**

Documentation https://safrazik.com/vue-file-agent/docs/#slots says that there are slots file-preview-before and file-preview-after, however they do not exist. Would be really nice to have them

**Comment by safrazik** [comment]
Yes it is available in master branch, scheduled for next release. See issue: #73

**Comment by marinaglancy** [comment]
hi @safrazik , thanks for the reply. When is the next release going to be available? TIA

**Comment by hifisultonauliya** [comment]
hi @safrazik , thanks for the reply. When is the next release going to be available?
